arrayfire/library.py

```diff
@@ -493,9 +493,12 @@ class VARIANCE(_Enum):
 
 def _setup():
     import platform
```

Comment:
The platform is a build-in package https://docs.python.org/3/library/platform.html. Why is it not called on the top level of the file?